lib/seed_builder/valid/string.rb

```diff
@@ -31,6 +34,10 @@ def generate
 
       private
 
+      def email_str
+        "example@example.com"
```

Comment:
@zaru It's better? （I like this one.
https://github.com/zaru/seed_builder/pull/33/commits/470723859bafa3806bc1c4df55ebded63015ed1c